### ISSUE TITLE
This does not work on IE 11

### ISSUE THREAD

**Issue by KirillBorunov** [amiller-gh/currentScript-polyfill#2]
As others currentScript polyfills, this one also relies on the script.readyState and not working on IE 11.

**Comment by amiller-gh** [comment]
Its been a very long time since I touched this, but it looks I supported IE11 by grabbing the current script's URL from a stack trace to find the element: https://github.com/amiller-gh/currentScript-polyfill/blob/master/currentScript.js#L21

Did you test in-browser to confirm this does not work?

**Comment by KirillBorunov** [comment]
Hi. Yes I tested it. With this code:
```
<!doctype html>
<html lang="es">
	<head>
		<meta charset="utf-8">
		<script src="currentScript.js"></script>
	</head>
	<body>
		<script id="test2">
			console.log(document.currentScript);
		</script>
		<form id="hola">
			<script id="test">
				console.log(document.currentScript);
			</script>
		</form>
	</body>
</html>
```

EDGE/Internet Explorer 12 - OK
Internet Explorer 11 - returns null on console.
Internet Explorer 10 (emulated) - OK
Internet Explorer 9 (emulated) - OK
Internet Explorer 8 (emulated) - returns an int values (maybe the index on the scripts)

**Comment by haoqunjiang** [comment]
I think this polyfill only works for scripts with `src` attribute in IE11

**Comment by Schepp** [comment]
Should be fixed with the following pull-request: https://github.com/amiller-gh/currentScript-polyfill/pull/5

**Comment by dasa** [comment]
Nice! Looks like the README is now out of date though.

> An exceptionally slim (~310b minified) polyfill for document.currentScript in IE9-IE10

**Comment by amiller-gh** [comment]
Fixed https://github.com/amiller-gh/currentScript-polyfill/commit/0b6a6ff80f673df4a5de24aa58427b43290d0a2e

Thanks for the callout @dasa!

**Comment by Schepp** [comment]
@amiller-gh can we get the current version published to npm?

**Comment by josueochoa** [comment]
> Should be fixed with the following pull-request: #5

hey @Schepp, this doesn't seem to work on IE11 as of right now, here's my test:
https://josueochoa.com/currentScript.html

Returns `null` on both scripts.

**Comment by Schepp** [comment]
Thank you for reporting this. It seems as if Microsoft updated IE 11 in a way so that the error argument in `catch` becomes `undefined`. Therefore the code cannot parse out line number and character anymore. Must be a regression on Microsoft's side.
